src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
-export interface Figure {
-
+enum OfFiruers {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle'
```

Comment:
2. [CODE STYLE] - Enum constant names usually start with capital letter. Values can have any case that's necessary.